### ISSUE TITLE
Accessibility issue - repeated ID's in navigation

### ISSUE THREAD

**Issue by earleyj1** [blueriver/MuraCMS#2837]
When using the same name for a page (with a different parent), the same ID is used in the navigation. This results in an accessibility issue in that ID's are repeated. This fails WCAG 2,0 4.1.1

![image](https://user-images.githubusercontent.com/29611694/38036693-4494bc4c-3275-11e8-9a16-179812c2e418.png)
